### PR TITLE
Feature: Added cart 

### PR DESCRIPTION
**Pull request by Adarsh-Patel-Dev**
image

image

### CODE REVIEW

File: src/components/CartPrice/cartPrice.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    font-weight: 400;
+    padding: 1rem;
+}
+
+.bold>*{
+    font-weight: 550;
+}
+
+.order-btn{
+    background-color: var(--BLUE-80) !important;
```
Comment (sri-rishi) [inline src/components/CartPrice/cartPrice.css]: Suggestion: We should not use `!important` as mentioned by Vikram

File: src/pages/cartPage/CartPage.jsx [hunk 2]
```diff
@@ -63,30 +66,7 @@ const CartPage = () => {
...
-            <hr/>
-            <div class="price-table bold">
-                <div> TOTAL AMOUNT </div>
-                <div> {'\u20B9'} 3999 </div>
-            </div>
-            <hr/>
-            <p class="price-component-para">You will save &#8377 500 on this order </p>
-            <button class="btn btn--primary order-btn">Proceed to Checkout</button>
-        </div>
+        { cart.length === 0 ? <h3 class="cart-heading-empty">Your cart is empty. Please add items in your <FaShoppingCart className="green"/></h3> : <CartPrice qty={qty} price={price} discountPrice={discountPrice} totalPrice={totalPrice}/>}
```
Comment (sri-rishi) [inline src/pages/cartPage/CartPage.jsx]: Suggestion: I think you forgot to replace `class` to `className` here

File: src/pages/logoutPage/Logout.jsx [hunk 3]
```diff
@@ -9,22 +8,16 @@ const Logout = () => {
...
-                <input type="checkbox" className="signup" />
-                <label className="signup-label">I agree to the Terms of Service and Privacy Policy</label>
-            </div> */}
             <span className="signup-span">You are Logged Out </span>
-
+            <Link to ="/login">
             <button href="#" className="btn-login">Login</button>
+            </Link>
+            <Link to ="/signup">
             <button href="#" className="btn-login outline">Signup</button>
```
Comment (sri-rishi) [inline src/pages/logoutPage/Logout.jsx]: Suggestion: if you are using Link then I think `href` attribute has any use here

**Review by sri-rishi — APPROVED** [review]
Nice work, just some suggestions which I think can be done otherwise feel free to merge

**Review by rdthetha — COMMENTED** [review]
Looks good to merge!
Can work on the suggestions by @sri-rishi !
Adding deployed site would be better to check the functionality!

File: src/components/CartPrice/CartPrice.jsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        </div>
+        <div class="price-table">
+            <div> Discount </div>
+            <div> - {'\u20B9'} {discountPrice} </div>
+        </div>
+        <div class="price-table">
+            <div> Delivery Charges </div>
+            <div> {'\u20B9'} 199 </div>
+        </div>
+        <hr/>
```
Comment (Sahil25061999) [inline src/components/CartPrice/CartPrice.jsx]: convert class to className

File: src/pages/cartPage/CartPage.jsx [hunk 5]
```diff
@@ -29,17 +31,18 @@ const CartPage = () => {
     console.log("my cart has",cart);
```
Comment (Sahil25061999) [inline src/pages/cartPage/CartPage.jsx]: remove console.log if not required.

**Review by Sahil25061999 — APPROVED** [review]
suggested some minor changes, the code looks good, good naming conventions have been used, can be merged after the suggested changes.

**Comment by Adarsh-Patel-Dev** [comment]
completed All P0 features of E-commerce

**Pull request merged**